ipd/0015/README.md

```diff
@@ -67,38 +67,38 @@ issue.
    for bhyve since it uses a dynamic sdev plugin to manage entries within
    /dev/vmm/
 
-1. [OS-6549](https://smartos.org/bugview/OS-6549) vmm segment driver
+2. [OS-6549](https://smartos.org/bugview/OS-6549) vmm segment driver
```

Comment:
Can you leave these all as `1.` ? Markdown takes care of numbering when it's rendered and it's much easier to add/remove/re-order things this way. (just for this list of pre-reqs, not the main section titles)

Reply:
I'm a fan of the markdown "rendering" roughly the same in the terminal as it does on github.

Reply:
Fair enough.